console/console-init/ui/src/modules/address/containers/AddressDefinitionContainer/AddressDefinitionContainer.tsx

```diff
@@ -106,9 +106,9 @@ export const AddressDefinitionContainer: React.FunctionComponent<IAddressDefinit
             description: plan.spec.shortDescription || plan.spec.longDescription
           };
         });
-        setPlan(" ");
         setTopic(" ");
         setPlanOptions(planOptions);
+        setPlan(" ");
```

Comment:
is there any reasoing of passing space in states functions?

Reply:
Yeah, I have not checked for this PF version, but earlier we had to give a space other wise the dropdown toggle was getting misplaced.